target/linux/kirkwood/files-5.4/arch/arm/boot/dts/kirkwood-linksys-audi.dts

```diff
@@ -165,22 +165,22 @@
 
 		partition@200000 {
 			label = "kernel1";
-			reg = <0x200000 0x290000>;
+			reg = <0x200000 0x300000>;
 		};
 
 		partition@490000 {
 			label = "rootfs1";
-			reg = <0x490000 0x1170000>;
+			reg = <0x500000 0x1160000>;
 		};
 
 		partition@1600000 {
 			label = "kernel2";
-			reg = <0x1600000 0x290000>;
+			reg = <0x1600000 0x300000>;
 		};
 
 		partition@1890000 {
 			label = "rootfs2";
-			reg = <0x1890000 0x1170000>;
+			reg = <0x1900000 0x1160000>;
```

Comment:
Same as above. Despite, are we actually doing something with the second set of partitions? If not, I wouldn't touch them.